gemviz/chartview.py

```diff
@@ -79,19 +84,24 @@ def __init__(self, parent, **kwargs):
             QtWidgets.QSizePolicy.Preferred, QtWidgets.QSizePolicy.Preferred
         )
 
-        self.plot_widget = pg.PlotWidget()
-        self.plot_widget.addLegend()
-        self.plot_widget.plotItem.showAxes(True)
-        self.plot_widget.plotItem.showGrid(x=True, y=True, alpha=GRID_OPACITY)
-        # see: https://stackoverflow.com/a/70200326
-        label = pg.LabelItem(
-            f"plot: {datetime.datetime.now()}", color="lightgrey", size="8pt"
-        )
-        label.setParentItem(self.plot_widget.plotItem)
-        label.anchor(itemPos=(0, 1), parentPos=(0, 1))
+        # Create a Matplotlib figure and canvas
+        self.figure = Figure()
+        self.canvas = FigureCanvas(self.figure)
+        self.main_axes = self.figure.add_subplot(111)
+        # Adjust margins
+        self.figure.subplots_adjust(bottom=0.1, top=0.9, right=0.92)
+        # self.figure.tight_layout()
+        self.setOptions()
+
+        # https://stackoverflow.com/questions/16066695/add-an-extra-information-in-a-python-plot
+        # https://matplotlib.org/stable/gallery/text_labels_and_annotations/annotation_demo.html
+
+        # now = datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+        # self.figure.text(0, 0, f"plot: {now}", ha="left", size=8, color="lightgrey")
```

Comment:
We don't need this merged into main

Reply:
agreed.  I paid for line 100 with a few hours of search time.